src/sonic-yang-models/yang-models/sonic-types.yang

```diff
@@ -114,4 +118,53 @@ module sonic-types {
             pattern "percentage|used|free|PERCENTAGE|USED|FREE";
         }
     }
+
+    typedef oper-status {
+        type enumeration {
+            enum unknown;
+            enum up;
+            enum down;
+        }
+        description "Operational status of an entity such as Port, MCLAG etc";
+    }
+
+    typedef mode-status {
+        type enumeration {
+            enum enable;
+            enum disable;
+        }
+        description
+            "This type can be used where toggle functionality required.
+            For ex. IPv6-link-local-only, Dhcp-replay-link-select, SNMP traps etc";
+    }
+
+    typedef dhcp-relay-policy-action {
+        type enumeration {
+            enum discard;
+            enum append;
+            enum replace;
```

Comment:
is replace also a yang statement

Reply:
Here "replace" is an enum. It is not a yang statement but it is an argument of deviate statement which is sub-statement of deviation statement.